factory/django.py

```diff
@@ -142,6 +143,10 @@ def _get_or_create(cls, model_class, *args, **kwargs):
         try:
             instance, _created = manager.get_or_create(*args, **key_fields)
         except IntegrityError as e:
+
+            if cls._original_params is None:
+                raise e
+
```

Comment:
'Great' minds think alike, except @francoisfreitag explained in issue #979 that it is not so great because " `_original_params` distinguishes the user lookup (for use in `.get()`) from the values generated by the factory (for use in `.create()`)". 

Thanks for your feedback, really appreciate it!